### PR TITLE
Commit for Sasha

### PR DESCRIPTION
**Pull request by V0S7ER**

### CODE REVIEW

File: NetMetods.py [hunk 1]
```diff
@@ -29,43 +28,35 @@ def train_epoch(model: nn.Module, optimizer: optim, scheduler: optim.lr_schedule
...
-        wave = model(mixture)
-        loss = loss_fn(wave, clean)
+    train_prob = 0
+
+    for mixture, target in tqdm(data_loader):
+        logits = model(mixture)
+        loss = loss_fn(logits, target)
         loss.backward()
+
+        prob = nn.functional.sigmoid(logits)
```
Comment (alexnikko) [inline NetMetods.py:39]: replace by torch.sigmoid()

File: NetMetods.py [hunk 2]
```diff
@@ -29,43 +28,35 @@ def train_epoch(model: nn.Module, optimizer: optim, scheduler: optim.lr_schedule
...
+        prob = (prob > 0.5).int()
```
Comment (alexnikko) [inline NetMetods.py:40]: float()

File: NetMetods.py [hunk 3]
```diff
@@ -29,43 +28,35 @@ def train_epoch(model: nn.Module, optimizer: optim, scheduler: optim.lr_schedule
...
+        train_prob += torch.mean(prob == target)  # TODO: тут питон ругается
```
Comment (alexnikko) [inline NetMetods.py:41]: .item()

File: NetMetods.py [hunk 4]
```diff
@@ -82,34 +73,23 @@ def val_epoch(model, data_loader, loss_fn, point: int, gl_point: int):
...
-                           mixture.detach().cpu().numpy() - clean.detach().cpu().numpy()).mean()
-        val_snr += cur_snr
-        val_inp_snr += inp_snr
+    val_prob = 0
+    for mixture, target in data_loader:
+        logits = model(mixture)
+        loss = loss_fn(logits, target)
         val_loss += loss.item()
-        val_snr_i += (cur_snr - inp_snr)
+        prob = nn.functional.sigmoid(logits)
```
Comment (alexnikko) [inline NetMetods.py:82]: torch.sigmoid()

File: NetMetods.py [hunk 5]
```diff
@@ -82,34 +73,23 @@ def val_epoch(model, data_loader, loss_fn, point: int, gl_point: int):
...
+        prob = (prob > 0.5).int()
```
Comment (alexnikko) [inline NetMetods.py:83]: float()

File: NetMetods.py [hunk 6]
```diff
@@ -82,34 +73,23 @@ def val_epoch(model, data_loader, loss_fn, point: int, gl_point: int):
...
+        val_prob += torch.mean(prob == target)  # TODO: тут питон ругается
```
Comment (alexnikko) [inline NetMetods.py:84]: float(torch.mean(...)) or .item()
